npm/src/index.ts

```diff
@@ -121,9 +110,12 @@ export const renderModules = async () => {
     } else {
       errorHandlingModule.handleUploadError(frontEndInfo)
     }
-  } else if (timeoutDialog) {
-    const sessionTimeoutModule = await import("./auth/session-timeout")
-    await sessionTimeoutModule.initialiseSessionTimeout()
+  }
+  if (!uploadContainer && !fileChecksContainer) {
```

Comment:
The `timeoutDialog`is no longer appearing on the upload page. I think it has something to do with this change. Was there a reason you added the `if(!uploadContainer && ! fileChecksContainer)`?

Reply:
Yes, when on the uploading page if the dialog appears and the 5 minutes has passed or upload succeeds. The prompt Are you sure you want to navigate _away from this page_ appears preventing the page from redirecting unless a user interacts. I tried to disable this no avail.

Reply:
This is the current behaviour on intg. So we should add the session dialog box to the pages. The important thing is that a user is logged out if they do not confirm they want to remain logged in.